core/src/main/java/bio/terra/pearl/core/model/kit/KitRequestStatus.java

```diff
@@ -17,6 +17,7 @@ public enum KitRequestStatus {
     CREATED,
     QUEUED,
     SENT,
+    COLLECTED, // for a kit that has been handed back to study staff in person. staff will mail it back to GP
```

Comment:
COLLECTED_BY_STAFF?

Reply:
If we're keeping track of the fact that it's sent via mail vs in person elsewhere, what is the distinction between this and `RECEIVED`? I would personally lean towards only have one option that denotes "We got it back from the participant" so that future if/switch statements are less complex/error prone, but I could go either way.

Reply:
`RECEIVED` currently means "received by GP"

`COLLECTED` represents a new intermediate step, eventually a kit in this state would progress to `RECEIVED` once it's sent back to GP by study staff

the current statuses are a bit confusing especially now that the kit lifecycle is becoming more complex, I wonder if it would be worthwhile to migrate/refactor the existing statuses to make them more descriptive

some additional comments on here would be useful too